pkg/lightning/config/config.go

```diff
@@ -1107,13 +1112,20 @@ type TikvImporter struct {
 	// default is PausePDSchedulerScopeTable to compatible with previous version(>= 6.1)
 	PausePDSchedulerScope PausePDSchedulerScope `toml:"pause-pd-scheduler-scope" json:"pause-pd-scheduler-scope"`
 	BlockSize             ByteSize              `toml:"block-size" json:"block-size"`
+
+	ChunkSize     int    `toml:"chunk-size" json:"chunk-size"`
+	ChunkCacheDir string `toml:"chunk-cache-dir" json:"chunk-cache-dir"`
```

Comment:
lightning already have a concept named `chunk` which means a source data file or part of it, what you mean it here?